tests/test_utils.py

```diff
@@ -13,6 +16,45 @@
 a = da.from_array(np.array([0, 1, 2, 3, 0]), chunks=3)
 X, y = make_classification(chunks=2)
 
+Foo = namedtuple('Foo', 'a_ b_ c_ d_')
+Bar = namedtuple("Bar", 'a_ b_ d_ e_')
+
+
+def assert_estimator_equal(left, right, exclude=None, **kwargs):
+    """Check that two Estimators are equal
+
+    Parameters
+    ----------
+    left, right : Estimators
+    exclude : str or sequence of str
+        attributes to skip in the check
+    kwargs : dict
+        Passed through to the dask `assert_eq` method.
+
+    """
+    left_attrs = [x for x in dir(left) if x.endswith('_')
+                  and not x.startswith('_')]
+    right_attrs = [x for x in dir(right) if x.endswith('_')
+                   and not x.startswith('_')]
+    if exclude is None:
+        exclude = set()
+    elif isinstance(exclude, str):
+        exclude = {exclude}
+    else:
+        exclude = set(exclude)
+
+    assert (set(left_attrs) - exclude) == set(right_attrs) - exclude
+
+    for attr in set(left_attrs) - exclude:
+        l = getattr(left, attr)
+        r = getattr(right, attr)
+        if isinstance(l, (np.ndarray, da.Array)):
+            assert_eq_ar(l, r, **kwargs)
+        elif isinstance(l, (pd.core.generic.NDFrame, dd._Frame)):
```

Comment:
Any reason why we can't compare against `pd.DataFrame` instead of `pd.core.generic.NDFrame`?
Same for `dd.DataFrame` and `dd._Frame`.